core/shared/src/main/scala/zio/concurrent/ConcurrentMap.scala

```diff
@@ -39,6 +92,14 @@ final class ConcurrentMap[K, V] private (private val underlying: ConcurrentHashM
   def put(key: K, value: V): UIO[Option[V]] =
     UIO(Option(underlying.put(key, value)))
 
+  /**
+   * Adds all new key-value pairs
+   */
+  def putAll(keyValues: (K, V)*): UIO[Unit] =
+    UIO(keyValues.foreach { case (k: K, v: V) =>
```

Comment:
It would be great if we could use `putAll` here. Would this work: `underlying.putAll(keyValues.toMap.asJava)`?

Reply:
I was trying to avoid using JavaConverters (https://github.com/zio/zio/pull/5933#discussion_r739778002). I see we used it in the methods on ConcurrentSet, so I switched it over and just added a silencer.